outputs.tf

```diff
@@ -5,8 +5,32 @@
 output "security_group" {
   description = "The security group attributes."
   value = try(
-    aws_security_group.self[0],
-    aws_security_group.self_with_name_prefix[0],
+    {
+      arn                    = aws_security_group.self[0].arn
```

Comment:
the solution is to explictly output the security group attributes that we want. excluding `ingress` and `egress`